src/systems/events/spawn_audio.rs

```diff
@@ -0,0 +1,27 @@
+use bevy::{
+    asset::AssetServer,
+    audio::AudioBundle,
+    ecs::{
+        event::EventReader,
+        system::{Commands, Res},
+    },
+};
+
+use crate::events::spawn_audio_event::SpawnAudioEvent;
+
+#[allow(dead_code)]
+pub fn spawn_audio(
+    mut spawn_audio_events: EventReader<SpawnAudioEvent>,
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+) {
+    for spawn_audio_event in spawn_audio_events.read() {
+        //If passed entity exists
+        if let Some(mut entity) = commands.get_entity(spawn_audio_event.entity) {
```

Comment:
Not sure why this doesn't work...

Reply:
you need to register the event in game_events_plugins